release_notes.md

```diff
@@ -12,3 +14,4 @@
 - Update PowerShell 7.4 worker to [4.0.4021](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.4021)
 - Updated dotnet-isolated worker to [1.0.10](https://github.com/Azure/azure-functions-dotnet-worker/pull/2629) (#10340)
 - Update Java Worker Version to [2.15.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.15.0)
```

Comment:
Do we need the 2.15 entry here since the version you are using is 2.16? It's implicit that 2.16 is built on top of 2.15

Reply:
I thought the same but I liked idea of browsing from release notes to the specific tag for each version and also the specific changes each version brings.

Reply:
> I thought the same but I liked idea of browsing from release notes to the specific tag for each version and also the specific changes each version brings.

Yes, this was the intent